todo_app/data/trello_items.py

```diff
@@ -0,0 +1,62 @@
+import os
+from requests import get, put, delete, post
+
+from todo_app.models.Item import Item
+
+BOARD_ID = os.getenv('TRELLO_BOARD_ID')
+TO_DO_LIST_ID = os.getenv('TRELLO_TO_DO_LIST_ID')
+IN_PROGRESS_LIST_ID = os.getenv('TRELLO_IN_PROGRESS_LIST_ID')
+COMPLETED_LIST_ID = os.getenv('TRELLO_COMPLETED_LIST_ID')
+
+key = os.getenv('TRELLO_KEY')
+token = os.getenv('TRELLO_TOKEN')
```

Comment:
Not important but worth considering, why the case-change between these and the ids above?